src/features/notificationblock/feature.json

```
@@ -1,6 +1,6 @@
 {
   "title": "NotificationBlock",
-  "description": "Block a post's notifications",
+  "description": "Hide notifications from a post",
```

I think "activity" is actually more accurate here; "notifications" could be construed as referring to the note view you can open on posts. (Yes, that component is called `PostActivity` internally; no, it's not an accurate name.)

```suggestion
  "description": "Hide activity about any post",
```

---

I like to use "activity items," would that fit here?